scripts/main.js

```diff
@@ -6,11 +6,278 @@ function coolMenu() {
+  document.getElementById('smallPicMobile4').src = 'resources/yellowLaptop.png';
+
+  if (window.screen.width < 768) {
+    mobileModal.classList.toggle('dispNone');
+  } else {
+    deskModal.classList.toggle('dispNone');
+  }
+}
+
+function openModal5() {
+  const deskModal = document.getElementById('modalDesktopID');
+  const mobileModal = document.getElementById('modalMobileID');
+
+  document.getElementById('mobileProject').innerHTML = 'Microverse';
+  document.getElementById('desktopProject').innerHTML = 'Microverse';
+
+  document.getElementById('bigPicMobile').src = 'resources/grayLaptop.png';
+  document.getElementById('smallPicMobile1').src = 'resources/grayLaptop.png';
+  document.getElementById('smallPicMobile2').src = 'resources/grayLaptop.png';
+  document.getElementById('smallPicMobile3').src = 'resources/grayLaptop.png';
+  document.getElementById('smallPicMobile4').src = 'resources/grayLaptop.png';
+
+  document.getElementById('bigPicDesktop').src = 'resources/grayLaptop.png';
+  document.getElementById('smallPicDesktop1').src = 'resources/grayLaptop.png';
+  document.getElementById('smallPicDesktop2').src = 'resources/grayLaptop.png';
+  document.getElementById('smallPicDesktop3').src = 'resources/grayLaptop.png';
+  document.getElementById('smallPicDesktop4').src = 'resources/grayLaptop.png';
+
+  if (window.screen.width < 768) {
+    mobileModal.classList.toggle('dispNone');
+  } else {
+    deskModal.classList.toggle('dispNone');
+  }
+}
+
+function openModal6() {
+  const deskModal = document.getElementById('modalDesktopID');
+  const mobileModal = document.getElementById('modalMobileID');
+  document.getElementById('mobileProject').innerHTML = 'YouTube';
+  document.getElementById('desktopProject').innerHTML = 'YouTube';
+  document.getElementById('bigPicMobile').src = 'resources/pinkScreen.png';
+  document.getElementById('smallPicMobile1').src = 'resources/pinkScreen.png';
+  document.getElementById('smallPicMobile2').src = 'resources/pinkScreen.png';
+  document.getElementById('smallPicMobile3').src = 'resources/pinkScreen.png';
+  document.getElementById('smallPicMobile4').src = 'resources/pinkScreen.png';
+  if (window.screen.width < 768) {
+    mobileModal.classList.toggle('dispNone');
+  } else {
+    deskModal.classList.toggle('dispNone');
+  }
+}
+function closeDesktopModal() {
+  const deskModal = document.getElementById('modalDesktopID');
+  deskModal.classList.toggle('dispNone');
+}
+function closeMobileModal() {
+  const mobileModal = document.getElementById('modalMobileID');
+  mobileModal.classList.toggle('dispNone');
+}
+function seeLive() {
+  window.open('https://fdezcaminero.github.io/figmaPortfolio2023/');
+}
+function seeSource() {
+  window.open('https://github.com/fdezcaminero/figmaPortfolio2023');
+}
+
+coolMenu();
+loadProjects();
+openModal();
+openModal2();
+openModal3();
+openModal4();
+openModal5();
+openModal6();
+closeDesktopModal();
+closeMobileModal();
+seeLive();
+seeSource();
```

Comment:
- [ ] These functions should be called when a certain event happens so please make sure to place them in an event listener as it makes it very difficult to trace the flow of the code when they're all called at once arbitrarily. ✔